linodecli/cli.py

```diff
@@ -485,6 +485,18 @@ def handle_command(self, command, action, args):
         action
         """
 
+        if (
+            (command == "linodes" and action == "ips-list")
+            or (command == "firewalls" and action == "rules-list")
```

Comment:
If you want to use some Python hackery, this could become something like:

```python
if (command, action) in [("linodes", "ips-list"), ("firewalls", "rules-list")]:
```

This is more of a stylistic change so feel free to ignore it 🙂

Reply:
That's much cleaner. Thanks!